tensorflow/core/kernels/data/batch_dataset_op_test.cc

```diff
@@ -27,10 +27,12 @@ class BatchDatasetOpTest : public DatasetOpsTestBase {
       const std::vector<PartialTensorShape>& output_shapes,
       std::unique_ptr<OpKernel>* batch_dataset_op_kernel) {
     NodeDef node_def = test::function::NDef(
-        kNodeName, kOpName, {"input_dataset", "batch_size", "drop_remainder"},
-        {{"parallel_copy", parallel_copy},
-         {"output_types", output_types},
-         {"output_shapes", output_shapes}});
+        kNodeName, "BatchDatasetV2",
```

Comment:
this should use `OpName`

Reply:
`name_utils::OpName(BatchDatasetOp::kDatasettype)` will return `BatchDataset`. Will it be good to change it to be `Strings::StrCat(name_utils::OpName(BatchDatasetOp::kDatasettype), `V2`)`?

Reply:
Similar to `PaddedBatch` you should respect the value of `op_version_`. So it should be "BatchDataset" when `op_version_ == 1` and "BatchDatasetV2" when `op_version_ == 2`.